config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
Yeah. You're getting thrown off by his comment. I'm not suggesting that you use the Devise namespace, which would amount to the controller class being `Devise::OmniauthCallbacksController`. You just need to make sure that the namespace use in your routing config and in your controller is consistent. The `users/` part in `"users/omniauth_callbacks"` is telling Rails to look for a controller under the users namespace (`Users::OmniauthCallbacksController`) in app/controllers/users, but the rest of your code is set up without a namespace, which in this context is just a way to organize related classes. The easiest fix is to change the route to the following, which is how that video has it set up:

`devise_for :users, :controllers => { :omniauth_callbacks => "omniauth_callbacks" }